Userland/Libraries/LibGLSL/Lexer.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2020, Stephan Unverwerth <s.unverwerth@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Token.h"
+
+#include <AK/DeprecatedString.h>
```

Comment:
It would be preferable to not use DeprecatedString in entirely new code.

Reply:
Again, I based this on the existing LibJS code.